.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+name: Request a Package
+description: Request an RPM package to be included in the base image
+labels: [package-request]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this request!
+  - type: textarea
+    id: describe-bug
+    attributes:
+      label: Describe the package
+      description: Include why you feel this should be on the image
+      placeholder: Tell us what you need
+      value: "I'd like to request the package `vim` on the main image"
+    validations:
+      required: true
+  - type: dropdown
+    id: image
+    attributes:
+      label: Image
+      description: Which specific image do you want?
+      options:
+        - main
+        - nvidia
```

Comment:
Isn't nvidia going to be in a different repo? Wouldn't this instead be which specific distro? `silverblue`, `kinoite`, `all`, etc

Reply:
I was unsure what to put there so thanks, I'll fix it. 🗡️

Reply:
PTAL